bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -30,3 +30,7 @@
 .invisible {
   visibility: hidden;
 }
+
+.isTalking {
+  text-shadow: 0 0 .35rem #FFF;
```

Comment:
the idea with the text-shadow was to get the icon as well at the text below the button to be affected. I did this because someone mentioned having just the icon glowing wasn't very noticeable.